### PR TITLE
Homgry

### PR DESCRIPTION
**Pull request by JerryWester**
# About The Pull Request

Doubles the rate of hunger and thirst in exchange for a trait point. Doesn't account for `NO_HUNGER` or `NO_THIRST`. Also doesn't account for synths. Done at the request of CinKro#1001, [Suggestion #3393](https://discord.com/channels/908082639704064051/1040135231786721330)

## Why It's Good For The Game

homger

## A Port?

No

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added the "Hungry and Thirsty" negative quirk, grants one point and doubles the hunger/thirst modifier
/:cl:

### CODE REVIEW

**Comment by CinKro** [comment]
I love you I have wanted this for so long

File: modular_splurt/code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -4,3 +4,32 @@
 			to_chat(user, "<span class='warning'>You can't seem to force yourself up right now!</span>")
 			return
 	. = ..()
+
+/datum/species/handle_thirst(mob/living/carbon/human/H)
+	if(HAS_TRAIT(H, TRAIT_NOTHIRST))
+		return
+
+	// A lot of this is just copied and pasted from the handle_digestion proc
+	if (H.thirst > 0 && H.stat != DEAD && !HAS_TRAIT(H, TRAIT_NOHUNGER))
```
Comment (BongaTheProto) [inline modular_splurt/code/modules/mob/living/carbon/human/species.dm:13]: shouldn't it be TRAIT_NOTHIRST?

Comment (JerryWester) [reply modular_splurt/code/modules/mob/living/carbon/human/species.dm]: Yes, I am the stupid

File: modular_splurt/code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -4,3 +4,32 @@
...
+		else if(H.satiety < -MAX_SATIETY)
+			H.satiety = -MAX_SATIETY
+		else if(H.satiety < 0)
+			H.satiety++
+			if(prob(round(-H.satiety/40)))
+				H.Jitter(5)
+			thirst_rate = 3 * THIRST_FACTOR
+		*/
+		thirst_rate *= H.physiology.thirst_mod
+		H.adjust_nutrition(-thirst_rate)
```
Comment (BongaTheProto) [inline modular_splurt/code/modules/mob/living/carbon/human/species.dm:35]:
```suggestion
		H.adjust_thirst(-thirst_rate)
```

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

**Pull request merged**